zellij-tile/Cargo.toml

```diff
@@ -7,6 +7,7 @@ description = "A small client-side library for writing Zellij plugins"
 license = "MIT"
 
 [dependencies]
+ansi_term = "0.12.1"
```

Comment:
Can this be removed now that `zellij-tile-extra` exists?

Reply:
You are very quick :), yes it can